src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -38,6 +51,12 @@
     }
 }
 
+.bp-ColorPickerControl-highlight {
+    &.bp-is-open {
```

Comment:
Open doesn't quite seem accurate, maybe `bp-has-attention` would be more appropriate?

Reply:
Yes that makes sense in the context of the highlight div.